### ISSUE TITLE
Replace dependency to org.eclipse.osgi.services

### ISSUE THREAD

**Issue by HannesWell** [eclipse-acceleo/acceleo#127]
The bundle `o.e.osgi.services` is deprecated for removal as part of https://github.com/eclipse-equinox/equinox/issues/18 and already has been changed to be empty and only re-export the 'official' OSGi artifacts published to Maven-Central.
It should be replaced by imports of the desired `org.osgi.service.*` packages so that the runtime can choose a suitable provider independently of the bundle name.

In case of Acceleo it should mainly be removed from the acceleo.equinox.launcher feature. You could probably rely on dependencies being pulled in transitively.

**Comment by HannesWell** [comment]
@Kellindil could you please have a look at this? It should be relatively simple to resolve.

**Comment by Kellindil** [comment]
Hello @HannesWell ,

Sorry I was in holidays until today.

Unfortunately, I don't have an easy mean to test this modification. We plan on moving on from Acceleo 3 (that still depends on osgi.services) to a new implementation in the future so hopefully this won't be completely deprecated until then so we can avoid this change.

**Comment by HannesWell** [comment]
Understand. Thanks for the clarification.
The removal is scheduled to happen for the 2026-03 SimRel. 
It would be great if you would have removed the usage of that bundle until then, in whichever way. :)

**Comment by HannesWell** [comment]
> The removal is scheduled to happen for the 2026-03 SimRel.

FYI, the removal of that bundle is now proposed in
- https://github.com/eclipse-equinox/equinox/pull/1309

**Comment by HannesWell** [comment]
The feature `org.eclipse.acceleo.equinox.launcher`, that used to require the `org.eclipse.osgi.services` seems to be absent only since SimRel 2026-03. But it also seems like Acceleo has not contributed a new release to SimRel before since August 2024.
I don't see a removal in the history of the master branch in the features folder. But I do see that it's still on the `R3_7_maintenance` branch, which seems to have forked from master in February 2021 already.

> We plan on moving on from Acceleo 3 (that still depends on osgi.services) to a new implementation in the future

So I assume that's what happened and in the meantime Acceleo 4 is released, rendering Acceleo 3 and the `org.eclipse.acceleo.equinox.launcher` feature and thus the reference to the `org.eclipse.osgi.services` bundle obsolete.

Therefore I'll assume this can be considered completed. @Kellindil or @ylussaud let me know if this incorrect.
We'll then proceed with the final removal in Equinox.

**Comment by ylussaud** [comment]
Yes, the new Acceleo 4 has no dependency to `org.eclipse.osgi.services`. We are now focusing our efforts on the development of Acceleo 4 and the maintenance branch (3.7.x) will most likely stay as it is in the future. If you need to migrate your project, you can have a look at the [migration documentation](https://github.com/eclipse-acceleo/acceleo/blob/master/plugins/org.eclipse.acceleo.aql.doc/pages/index.adoc#migrate-acceleo-3-templates-to-acceleo-4).
